oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreBuilder.java

```diff
@@ -167,6 +169,8 @@ public class DocumentNodeStoreBuilder<T extends DocumentNodeStoreBuilder<T>> {
     private Predicate<Path> nodeCachePredicate = Predicates.alwaysTrue();
     private boolean clusterInvisible;
     private boolean throttlingEnabled;
+    private boolean fullGCEnabled;
```

Comment:
Not sure if it matters, but why don't we set it to the default value, too?

Reply:
We can set them, but the existing pattern is not to set them.

Reply:
That is true, but then I guess we shouldn't set the default for embeddedVerificationEnabled either.

Reply:
> That is true, but then I guess we shouldn't set the default for embeddedVerificationEnabled either.

It is set cause the default value for this is `true`; if not initialized, it would be set to `false`.

IIUC, this could be the reason for not setting the default for many booleans (whose default value is `false`) cause that would be assigned by JVM.

Reply:
Yeah, obviously, that wasn't a particularly bright comment of mine :)